api/src/main/resources/liquibase-update-to-2.3.xml

```diff
@@ -33,4 +33,16 @@
     	<renameColumn tableName="allergy" oldColumnName="comment" newColumnName="comments" columnDataType="varchar(1024)"/>
     </changeSet>
 
+    <changeSet id="TRUNK-5413" author="odorajonathan" dbms="mysql">
+      <preConditions onFail="MARK_RAN">
+            <not><columnExists tableName="concept" columnName="category"/></not>
+      </preConditions>
+      <comment>
+          Add Category field to OrderSet
+      </comment>
+      <addColumn tableName="concept">
+            <column name="category" type="char(38)"/>
```

Comment:
That's one of the changes I made for the types, after trying other types too. I tried looking at a couple of changesets like those from liquibase 2.0 for any changesets of concept, seemed to have the same types tht cause me the errors

Reply:
Can you point me to an example concept changeset that you are looking at?

Reply:
This [LINE ](https://github.com/openmrs/openmrs-core/blob/4a0a626df29ca255d7c18d3a606e85fdde3f5e04/api/src/main/resources/liquibase-update-to-2.0.xml#L799) and this [LINE](https://github.com/openmrs/openmrs-core/blob/4a0a626df29ca255d7c18d3a606e85fdde3f5e04/api/src/main/resources/liquibase-update-to-2.0.xml#L1174)

Reply:
I do not mean the concept table. I mean a column which is a concept.

Reply:
ok, looks like I have been looking wrongly, I kept noticing concept under table rather than as a column. Let me r-look through again

Reply:
@dkayiwa I haven't been able to notice a column name as plainly concept, most columns are concept_id or concept_name_id such like [this](http://cc.bingj.com/cache.aspx?q=<column+name%3d"concept"+liquibase-update-2.0+openmrs+github&d=4910090098772928&mkt=en-WW&setlang=en-US&w=BAWaW6TrHHi42eUE6x2PuhApCB2IXGki), could you guide

Reply:
The name does not matter. All that matters is a column that stored a concept.

Reply:
Look for a column that stores the same data type as the one you are working on.